velox/dwio/dwrf/reader/ReaderBase.cpp

```diff
@@ -21,8 +21,19 @@
 #include "velox/common/process/TraceContext.h"
 #include "velox/dwio/common/Mutation.h"
 #include "velox/dwio/common/exception/Exception.h"
+#include "velox/functions/lib/string/StringImpl.h"
 
 namespace facebook::velox::dwrf {
+namespace {
+
+// Perform lower for utf8 string input.
+std::string utf8StrToLowerCopy(const std::string& str) {
```

Comment:
Can you create one function for this and call it in different readers?

Reply:
Thanks for helping review. I added a `utf8StrToLowerCopy` function  in the 'velox/functions/lib/string/StringImpl.h'.